components/list/list-item-generic-layout.js

```diff
@@ -197,20 +205,24 @@ class ListItemGenericLayout extends RtlMixin(LitElement) {
 		return html`
 			<slot name="control-container"></slot>
 			<slot name="outside-control-container"></slot>
+
 			<slot name="drop-target"></slot>
-			<slot name="content-action" class="d2l-cell" data-cell-num="6"></slot>
-			<slot name="outside-control-action" class="d2l-cell" data-cell-num="1"></slot>
-			<slot name="outside-control" class="d2l-cell" data-cell-num="2"></slot>
-			<slot name="control-action" class="d2l-cell" data-cell-num="3"></slot>
-			<slot name="expand-collapse" class="d2l-cell" data-cell-num="4"></slot>
-			<slot name="control" class="d2l-cell" data-cell-num="5"></slot>
-			<slot name="actions" class="d2l-cell" data-cell-num="7"></slot>
-			<slot name="content" class="d2l-cell" data-cell-num="8" @focus="${!this.noPrimaryAction ? this._preventFocus : null}"></slot>
+
+			<slot name="content-action" class="d2l-cell" data-cell-num="0"></slot>
```

Comment:
Original order...
```
1 ::slotted([slot="outside-control-action"]) {
2 ::slotted([slot="expand-collapse"]) {
3 ::slotted([slot="control-action"]) {
4 ::slotted([slot="content-action"]) {
5 ::slotted([slot="actions"]) {
6 ::slotted([slot="drop-target"]) ➡️  now using `z-index: 1`
7 :host(.d2l-dragging-over) ::slotted([slot="nested"]) ➡️ now using `z-index: 2`
```
With these changes, it looks like `content-action` is being moved to be underneath everything, where is was previously on top of `control-action`. Does that cause issues?

Reply:
Also the `z-index` for drag-n-drop probably aren't necessary anymore either.

Reply:
Awesome. Thanks for the explanation. 😄